.prettierrc.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  trailingComma: "es5",
+  arrowParens: "avoid",
+  printWidth: 80,
+  tabWidth: 2,
+  useTabs: false,
+  semi: true,
+  quoteProps: "as-needed",
+  jsxSingleQuote: false,
+  singleQuote: false,
+  bracketSpacing: true,
+  bracketSameLine: false,
+  proseWrap: "always",
+  endOfLine: "lf",
+};
```

Comment:
Refer to other packages, use common configs & assign this PR back to me.
@anmol-kumar0815 _a

Reply:
I did not use common config here because common config includes [tailwindConfig](https://github.com/bigbinary/neeto-commons-frontend/blob/main/configs/prettier.js#L2) and here we do not need it. It will always give a warning.